gst/tensor_reposink/tensor_reposink.c

```diff
@@ -186,6 +181,7 @@ gst_tensor_repopush_set_property (GObject * object, guint prop_id,
       break;
     case PROP_SLOT:
       self->myid = g_value_get_uint (value);
+      gst_tensor_repo_add_repodata (self->myid);
```

Comment:
Be aware of dynamism.
PROP_SLOT may get different values in run-time. (changed by user / GUI tools)

Reply:
I think it is possible with proper handling the memory slot and lock/cond in tensor repository. Next time we could do to support this dynamism.